src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1257,6 +1257,8 @@ internal void LoadNodeLibrary(Assembly assem)
             if (!NodeModelAssemblyLoader.ContainsNodeModelSubType(assem))
             {
                 LibraryServices.LoadNodeLibrary(assem.Location, false);
+                LibraryServices.OnLibrariesImported(new LibraryServices.LibraryLoadedEventArgs(new List<string> { assem.Location }));
```

Comment:
I think it should simply be changed from:
```
LibraryServices.LoadNodeLibrary(assem.Location, false);
```
to:
```
LibraryServices.ImportLibrary(assem.Location);
```
as was the case before:
https://github.com/DynamoDS/Dynamo/pull/9736/files#diff-c1fbd0022486c5a40f24b0c5e51338e5L1255

Reply:
ok, will try that and rerun self serve.